libs/libloggers/loggers/OwnSplitChannel.cpp

```diff
@@ -20,7 +20,9 @@ void OwnSplitChannel::log(const Poco::Message & msg)
     if (channels.empty() && (logs_queue == nullptr || msg.getPriority() > logs_queue->max_priority))
         return;
 
-    if (auto masker = sensitive_data_masker.load())
+    std::lock_guard<std::mutex> lock(masker_mutex);
+
+    if (auto masker = sensitive_data_masker.lock())
```

Comment:
Do we apply an unknown number of regexes to every log message while holding a server-wide mutex?